dist/releases/4.12/config.toml

```diff
@@ -37,3 +37,7 @@ files = ["/usr/bin/cpb"]
 [[payload.operator-lifecycle-manager-container.ignore]]
 error = "ErrLibcryptoMissing"
 files = ["/usr/bin/cpb"]
+
+[[payload.multicluster-operators-subscription-operator-container.ignore]]
+error = "ErrLibcryptoSoMissing"
+files = ["/policy-generator/PolicyGenerator-rhel8"]
```

Comment:
@gparvin do you have a link to where or how this container is built?

Does is do any sort of cryptographic operations?

Reply:
We build the flagged binary with the OpenShift golang builder for RHEL 8: `openshift-golang-builder:v1.21.11-202407021847.g1ac3e39.el8` so that it will be FIPS ready if it is copied to a RHEL 8 FIPS enabled container.  
Since this binary is not intended for use in the multicluster subscription operator RHEL 9 container we get the error shown.  The build environment uses CPaaS so i can send you that info if you really need it.  The repository the policy-generator is built from is: https://github.com/stolostron/policy-generator-plugin.
Thanks